src/sentry/monitors/endpoints/monitor_ingest_checkin_details.py

```diff
@@ -70,7 +70,14 @@ def put(
             return self.respond(status=400)
 
         serializer = MonitorCheckInValidator(
-            data=request.data, partial=True, context={"project": project, "request": request}
+            data=request.data,
+            partial=True,
+            context={
+                "project": project,
+                "request": request,
+                "monitor_slug": monitor.slug,
+                "monitor": monitor,
```

Comment:
For if they're sending monitor config right?

Is there a test we can add to cover this?

Reply:
I can add a test but it really would just validate the bugfix. We aren't doing anything with the config yet (and not sure we want to).